src/components/VaccinationBlock/VaccinationBlock.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Region, MetroArea } from 'common/regions';
+import { getVaccineInfoByFips } from 'cms-content/vaccines/phases';
+import RegionVaccinationBlock from 'components/RegionVaccinationBlock';
+import VaccinationEligibilityBlock from 'components/VaccinationEligibilityBlock';
+import ErrorBoundary from 'components/ErrorBoundary';
+
+const VaccinationBlock: React.FC<{ region: Region }> = ({ region }) => {
+  const vaccineInfo = getVaccineInfoByFips(region.fipsCode);
```

Comment:
I think this isn't working for counties.

Perhaps we can make `getVaccineInfo()` automatically map county=>state and return the appropriate info?  And then it can probably do the same for metro=>state and do the "multi state metro" logic to return `null` in that case?